Packages/ohif-measurements/client/components/measurementLightTable/measurementLightTableView/measurementLightTableView.js

```diff
@@ -0,0 +1,39 @@
+import { Template } from 'meteor/templating';
+import { Tracker } from 'meteor/tracker';
+import { ReactiveVar } from 'meteor/reactive-var';
+import { _ } from 'meteor/underscore';
+import { OHIF } from 'meteor/ohif:core';
+
+Template.measurementLightTableView.onCreated(() => {
+    const instance = Template.instance();
+    const { measurementApi, timepointApi } = instance.data;
+
+    instance.data.measurementGroups = new ReactiveVar();
+
+    Tracker.autorun(() => {
```

Comment:
Don't use Tracker.autorun. It isn't destroyed when the template is destroyed. Use instance.autorun instead.